apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -290,18 +290,21 @@ function ListBoxInline({ options, layout }) {
                       onClick={unlock}
                       disabled={selectDisabled()}
                       size="large"
+                      sx={{ '&.Mui-disabled': { color: '#000', opacity: 0.9 } }}
```

Comment:
Can we get this from the theme?
If you change the background color in the theme to #000, you won't be able to change this to #FFF.

Reply:
Refactor so we don't need to change color and opacity for icons in edit mode.